### DISCUSSION TITLE
joining the table

### DISCUSSION THREAD

**Discussion by legaltextai** [freelawproject/courtlistener#3116] [Q&A]
Do I take it that in order to get in one table 
1. name of the case 
2. date of the case
3.  text of the opionion 
I need to join "opinions" with "dockets" on "id"? 
Thanks

**Comment by mlissner** [comment]
I recommend you download the schema from the bulk data or look at the diagram of it in the API documentation, but in general you'll probably want to join:

 - Courts to
 - Dockets to
 - Clusters to
 - Opinions

That will usually do the trick.

**Reply by legaltextai** [reply]
thanks. but do i join them on which field? "id" ? 
i tried to look through the picture of the schema , but it 's not very easy , or my screen is just too small ...

**Reply by mlissner** [reply]
Yeah, the schema file is huge. Usually, yes, it will be an id field and an `_id` field.
